### ISSUE TITLE
Telomere down

### ISSUE THREAD

**Issue by kmcgregor123456** [kmcgregor123456/CVNRimagingCore#17]
SMART failures on hard disk. Restoring our DICOMs from BACKUP.

**Comment by kmcgregor123456** [comment]
Telomere is being rebuilt now. Smaller hard disk at 500GB. Data transfers MUST BE REMOVED after exporting for scrubbing

**Comment by kmcgregor123456** [comment]
Test transfer successful

**Comment by kmcgregor123456** [comment]
Fixed. OK this is a dumb fix.
For some reason, HOROS removed the option to use dcmtk to parse dicoms in the later releases of the software. However, the capability is still built into version 1.4.4. After you install 1.4.4 it retains a preference file that tells the program to use dcmtk. Then when you install 3.0.1 (latest release) the preference file is still there and it still uses dcmtk.

So here is the fix:
1) Reinstall the operating system (not necessary, but I had to do it anyway)
2) Download Horos 1.4.4 from the horos website and install
3) Open Horos 1.4.4 and make sure that USE DCMTK to parse DICOMS is selected in PREFERENCES->General
4) Download Horos 3.0.1 and mount the .dmg image
5) Close Horos 1.4.4
6) Copy Horos 3.0.1 into the Applications folder and REPLACE old Horos
7) Run Horos 3.0.1 and the parsing ability will still be there.
